src/plugins/gui/pluginUtils.ts

```diff
@@ -15,22 +15,21 @@ export const createStore = (storeId: string, store: EdgeDataStore): FiatProvider
   }
 }
 
-// @ts-expect-error
-const ERROR_PRIORITIES: { [errorType: FiatProviderQuoteErrorTypes]: number } = {
-  underLimit: 1,
-  overLimit: 2,
-  paymentUnsupported: 3,
-  regionRestricted: 4,
-  assetUnsupported: 5
-}
+const ERROR_PRIORITIES = new Map<FiatProviderQuoteErrorTypes, number>([
```

Comment:
I don't understand how this commit is a cleanup. Logic just became more complicated. Fixing the `ts-expect-errors` could have been one-liners each.